src/main/java/com/github/onsdigital/babbage/api/filter/CorsFilter.java

```diff
@@ -14,7 +14,7 @@ public boolean filter(HttpServletRequest request, HttpServletResponse response)
 
         String requestType = URIUtil.resolveRequestType(request.getRequestURI());
 
-        if (requestType.equals("data")) {
+        if (requestType.equals("data") || request.getRequestURI().equals("/chartconfig")) {
```

Comment:
Will approve, but is there likely a need to add CORS to additional endpoints in the future? If so, this could become quite messy